pypeit/spectrographs/ntt_efosc2.py

```diff
@@ -0,0 +1,268 @@
+"""
+Module for NTT EFOSC2
+
+.. include:: ../include/links.rst
+"""
+import glob
+from pkg_resources import resource_filename
+
+import numpy as np
+
+from pypeit import msgs
+from pypeit import telescopes
+from pypeit.core import parse
+from pypeit.core import framematch
+from pypeit.spectrographs import spectrograph
+from pypeit.images import detector_container
+
+
+class NTTEFOSC2Spectrograph(spectrograph.Spectrograph):
+    """
+    Child of Spectrograph to handle NTT/EFOSC2 specific code
+    """
+    ndet = 1  # Because each detector is written to a separate FITS file
+    telescope = telescopes.NTTTelescopePar()
+    name = 'ntt_efosc2'
+    camera = 'EFOSC2'
+    comment = 'The ESO Faint Object Spectrograph and Camera version 2'
+
+    def configuration_keys(self):
+        """
+        Return the metadata keys that define a unique instrument
+        configuration.
+
+        This list is used by :class:`~pypeit.metadata.PypeItMetaData` to
+        identify the unique configurations among the list of frames read
+        for a given reduction.
+
+        Returns:
+            :obj:`list`: List of keywords of data pulled from file headers
+            and used to constuct the :class:`~pypeit.metadata.PypeItMetaData`
+            object.
+        """
+        return ['dispname', 'decker', 'binning']
```

Comment:
can the grating tilt?  If so, include the tilt angle here.

Reply:
According to my experience, there is no selection of tilt angles. Neither do I see a related parameter in the fits file.